source/leuven/services/openondemand.rst

```diff
@@ -0,0 +1,250 @@
+
+Templates
+~~~~~~~~~
+
+To enter the Templates menu, you can click on 'Templates' at the top once you are in the 'Job Composer' menu. You can also access this menu by clicking the button 'New 
+Job'-'From Template'. Once in this menu, you should see a table with 3 System Templates. The resources that are requested in these scripts are the default settings. 
+The templates:
+
+- CPU job template: this is a template for CPU jobs (batch partition). This is also the default template (which you will get when clicking 'From Default Template' under the 'New Job' button in the 'Jobs' menu).
+- GPU job template: a template for creating GPU jobs 
+- Big memory CPU jobs: a template to create job scripts that submit to the big memory partition instead of the default batch partition
+
+You can create your own templates from scratch or by copying one of the existing templates. In both cases you will be redirected to a page where you can provide a
+name, the cluster (only wICE) and add some notes. Once you save this, you will create a new folder in the templates folder 
+``$VSC_DATA/ondemand/data/sys/myjobs/templates/``. This information will be saved in the ``manifest.yml`` file that is always present in each of the template folders. 
+Which other files that are present in this folder will depend on how you created your new template. If you use the 'New Template' button, it will also contain the job 
+script of the default template, being the CPU job template. If you use the 'Copy Template' button, the contents of the folder will depend on the template you are
+copying from. Use the system template that suit your needs the best when starting out. Once you use this more often, you can also use your own templates to create new
+ones. Any file that is present in that folder, will be copied to your new one as well.
+
+Once you've created the new template folder, you can start customizing it to your needs. You could of course immediately create a new job from it, adapt some
+extra options and launch it (which will be explained in the next chapter), but you can still further costumize your template folder first. You can view the files in
+the folder using the Folder Explorer (click 'View Files' on top or 'Open Dir' at the bottom). As explained above, you can edit or remove any file, create new files,
+upload new files... These files will be present in each job you create from this template, which means you fully adapt this to the standard set-up you use for your
+jobs.
+
+Jobs
+~~~~
+
+The functioning of creating jobs is a bit similar to how you create new templates. Whatever method you choose, you will create a new folder for each job, this time
+located at ``$VSC_DATA/ondemand/data/projects/default/``. The job folders will be numbered according to the order you have created them in. **Do not change this folder name as long as you plan on using it from the job menus, as this will break the linking here.** When removing a job, the folder will be deleted as well. 
```

Comment:
```suggestion
located at ``$VSC_DATA/ondemand/data/projects/default/``. The job folders will be numbered in the order you have created them. **Do not change this folder name as long as you plan on using it from the job menus, as this will break the linking here.** When removing a job, the folder will be deleted as well. 
```